inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -8,27 +8,37 @@ To initialize the scenario, the user can define the following parameters:
 
 Now, the user is ready to simulate the steady state. Here, the user chooses the object to be inserted inside the wind tunnel. This object is defined with a geometry file in STL or OBJ format. Notice that, the required meshing step will automatically be made before starting the simulation. The meshing is done with the [snappyHexMesh tool](https://www.openfoam.com/documentation/user-guide/4-mesh-generation-and-conversion/4.4-mesh-generation-with-the-snappyhexmesh-utility) of OpenFOAM.
 
+To test this scenario we have available one example of a vehicle geometry in OBJ format - [download it here](../../../../resources/geometry/test_vehicle.obj). The user can also use his own geometry file, as long as it is in STL or OBJ format.
+
```

Comment:
Is it possible to have the path of the OBJ as absolute? 
If we change the folders in the future this can break

Reply:
I think that is better. Also, we may have a folder under www.inductiva.ai where we add resources. WDTY @hpenedones.

Finally, for a future demo, consider also using a randomly created object instead? Can we do it?

Reply:
> I think that is better. Also, we may have a folder under [www.inductiva.ai](http://www.inductiva.ai) where we add resources. WDTY @hpenedones.
> 
> Finally, for a future demo, consider also using a randomly created object instead? Can we do it?

I think that storing the example files here in the git repo is fine (unless they are too big, in that case we don't want the git repo to get too bloated). Having them hosted on our website would mean that we also have to update the website repo and redeploy it, when we make changes to the documentation.
But one thing we need to verify is that these links work when the README is showed in pypi.
https://pypi.org/project/inductiva/

It seems that we have a lot of broken links there when the user is directed to other scenarios, etc. @IvanPombo @luispcunha

Reply:
On it! I believe it is just need to address the main README. I will merge this PR and open a new one with those changes.